src/utils/kukai.utils.ts

```diff
@@ -17,6 +17,13 @@ const decrypt = async (chiphertext: string, password: string, salt: string, vers
   }
 };
 
+export class KukaiError extends Error {
+  constructor() {
+    super();
+    throw new Error(KUKAI_VERSION_ERROR);
```

Comment:
I'm not sure that throwing a new Error inside `constructor` is the right way to create a custom error.  
mb just set a correct message?
<img width="749" alt="image" src="https://user-images.githubusercontent.com/10281464/156029712-d7d1b3e8-0a7f-45be-8643-e873283d534e.png">

Reply:
but its typescript, not a esjs

Reply:
so, you could google solution for ts
https://ashsmith.io/handling-custom-error-classes-in-typescript
```
class MyError extends Error {
  constructor() {
    super('your message');
    Object.setPrototypeOf(this, MyError.prototype)
  }
}
```